proto/process/connections.proto

```diff
@@ -227,6 +227,9 @@ message Connection {
   // Meant to be used with CollectorConnections#encodedTags.
   // Handled the built-in race for resolution of container tags in the backend.
   int32 localContainerTagsIndex = 54;
+
+  // Indicates that this connection is from system-probe
+  bool systemProbeConn = 55;
```

Comment:
@AlexandreYang I realize we don't currently have a choice, but the downside to this is: this field only matters between system-probe and process-agent. If you have a way to prevent it from being sent to the backend (and thus increasing the size of payloads), that would be desirable.